CHANGELOG.md

```diff
@@ -6,10 +6,14 @@
 - Don't include changes that are purely internal. The CHANGELOG should be a
   useful summary for people upgrading their application, not a replication
   of the commit log.
-  
+
+## Unreleased
+
+* Make jasmine tests pass in Internet Explorer ([PR #2090](https://github.com/alphagov/govuk_publishing_components/pull/2090))
```

Comment:
Do we need to add this to changelog, as it's a purely internal change?

Reply:
I think so, because I believe it also fixes some bugs in IE10 (documented in the description).